TestControllers/AzureController.psm1

```diff
@@ -83,15 +76,21 @@ Class AzureController : TestController
 			}
 		}
 		elseif ($this.ARMImageName) {
-			&$ValidateARMImageName
+			$ArmImagesToBeUsed = @($this.ARMImageName.Trim(", ").Split(',').Trim())
+			if ($ArmImagesToBeUsed | Where-Object {$_.Split(" ").Count -ne 4}) {
```

Comment:
can we handle this situation when there are more than 1 space between publisher, offer and sku?

Reply:
yes, just moving previous code from Run-LISAv2 to AzureController, because the '-ARMImageName' is a special parameter for Azure only